### DISCUSSION TITLE
Add a tool to help finding chipped blade and amber larva

### DISCUSSION THREAD

**Discussion by ghost** [Ceddini/CoreKeeperMapTool#5] [Feature Requests]
these things are hard to find

**Comment by Ceddini** [comment]
Basically impossible. The map tool can only highlight / find things that are already visible on the map, i.e. if you have already found it. 
Both the chipped blade and amber larva spawn completely randomly. So the map doesn't know about it, unless it was already visible to you. 

Only thing that can be done is something like the maze highlighter. But that doesn't really help, since basically most black spaces in Azeos' Wilderness are valid spawn points for the temple. Same basically goes for the amber larva scene.

A small tip: Amber larva can now be farmed by defeating ghorm. So even if you're unable to find it, you can just kill ghorm over and over again. It's an extremely small chance though.
